tests/describe_task_queue.go

```diff
@@ -128,55 +142,82 @@ func (s *DescribeTaskQueueSuite) publishConsumeWorkflowTasksValidateBacklogInfo(
 			continue // poll again on empty responses
 		}
 		i++
+		_, err := s.client.RespondWorkflowTaskCompleted(NewContext(), &workflowservice.RespondWorkflowTaskCompletedRequest{
```

Comment:
any specific reason as to why we are responding with `RespondWorkflowTaskCompleted`?  IIRC, not responding with task completion should not impact if a poller picks up another task or not right?

Reply:
In RespondWorkflowTaskCompleted I'm sending an activity command which results in scheduling an activity task. I want to validate activity stats too.